packages/components/src/List/Toolbar/Pagination/Pagination.component.js

```diff
@@ -73,32 +74,32 @@ function Pagination({ id, startIndex, itemsPerPage, totalResults, onChange, ...o
 			>
 				{itemsPerPageOptions.map((option, index) => getMenuItem(option, index))}
 			</NavDropdown>
-			{itemsPerPage > 0 && (
+			{isNavigationShown && (
```

Comment:
Can you extract the navigation items to a function `getNavigationItems()` so you can write 
```
{isNavigationShown && getNavigationsItems()}
```
This would avoid the `sNavigationShown && ` on each items (5 times)